README.md

```diff
@@ -10,6 +10,14 @@ Konveyor Forklift controller.
 
 For the build, the forklift uses [Bazel](https://bazel.build/).
 
+Install the following system components:
+
+- bazel >= 5
+- docker
+- gcc
+- go
```

Comment:
Are you sure? What for? As far as I can tell bazel pulls in go toolset for the project. Specifically version 1.16.5 that we specify in `WORKSPACE`.